models/index.js

```diff
@@ -1 +1,14 @@
+const Hero = require('./Hero');
+const Mission = require('./Mission');
+const User = require('./User');
 
+Hero.hasMany(User, {
```

Comment:
Why does a hero have many accounts?

Reply:
Thought process was each user can see the Hero table, however I can revert it to a one to one relationship.